axlearn/common/struct_test.py

```diff
@@ -58,7 +58,7 @@ class SlotsPoint:
 
     def test_pytree_nodes(self):
         p = _Point(x=1, y=2, meta={"abc": True})
-        leaves = jax.tree_util.tree_leaves(p)
+        leaves = jax.jax.tree.leaves(p)
```

Comment:
Hm, this looks like a typo?
```suggestion
        leaves = jax.tree.leaves(p)
```
Likewise below.

Reply:
fixed